authorization/v2/uniformResourceIdentifier_test.go

```diff
@@ -42,3 +42,153 @@ func TestNewUriFromRequest(t *testing.T) {
 	}, uri.arguments)
 	assert.Nil(t, uri.metadata)
 }
+
+func Test_NewURIFromString__should_return_correct_URI(t *testing.T) {
```

Comment:
Just as a note for the future. You could write these tests as a table-driven test (like `Test_NewURIFromString__should_throw_error`). That might look a bit cleaner.

Don't worry about it now though, these are justs tests, it's fine.

Reply:
Oh that's true, I can refactor it quickly to use table-driven tests